pkgs/applications/misc/koreader/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, makeWrapper, dpkg, luajit, gtk3-x11, SDL2, glib, noto-fonts
+, nerdfonts, }:
+let font-droid = nerdfonts.override { fonts = [ "DroidSansMono" ]; };
+in stdenv.mkDerivation rec {
+  pname = "koreader";
+  version = "2020.09";
+
+  src = fetchurl {
+    url =
+      "https://github.com/koreader/koreader/releases/download/v${version}/koreader-${version}-amd64.deb";
+    sha256 = "12kiw3mw8g8d9fb8ywd4clm2bgblhq2gqcxzadwpmf0wxq7p0v8z";
+  };
```

Comment:
If you're fetching from github, it's probably better to use `fetchFromGitHub`. Any particular reason why you didn't?

Reply:
As far as I know, `fetchGitHub` only suports fetching a repo. I am fetching a release. I was not able to build koreader from the source file. For the time being, I just used the released deb file.

Reply:
Yeah that makes sense. This looks good to me.